learning-development/learning-support.qmd

```diff
@@ -48,23 +53,76 @@ In this workshop, we cover:
 
 Contact [statistics.development@education.gov.uk](mailto:statistics.development@education.gov.uk) to register interest in workshops happening at your site or to request new topics!
 
-## Partnership programme
+---
 
- *	Dedicated resource to support development work, either building skills or working on analytical or statistical outputs.
- *	A short-term project with an agreed set of goals and timescale decided as part of an initiation meeting.
- * Contact statistics.development@education.gov.uk if you're interested in finding out more about the support we can give.
+## Partnership programmes
 
-## Specific resources
+The Statistics Development Team offers bespoke partnership programmes for analyst teams across the department, designed as dedicated resource to support development work, either building skills or filling resource gaps to allow improvements in analytical or statistical outputs.
```

Comment:
Should this be SSU rather than SDT? thinking how we acknowledge partnerships Cam's team may help with (like the current FE dashboard one)

Reply:
I guess we don't want to commit Cam's team to loads of partnerships? Unless Cam wants loads of partnerships...?

Reply:
Agree with @lauraselby, I'd say SSU, saying that doesn't commit us to anything but opens it up so we can resource however we want to from the unit

Reply:
Have changed to SSU 😄